hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -174,6 +174,17 @@ public final class Hedera implements SwirldMain {
     private HandleThrottleAccumulator handleThrottling;
     private MonoMultiplierSources monoMultiplierSources;
 
+    /**
+     * The application name from the platform's perspective. This is currently locked in at the old main class name and
+     * requires data migration to change.
+     */
+    public static final String APP_NAME = "com.hedera.services.ServicesMain";
```

Comment:
Do we have an issue to change the name?

Reply:
I'm not aware of an issue. It will require data migration since this is currently used for file paths on disk. It will probably also need tooling updates, since I'm assuming there is probably a lot of tooling that expects the files to be in a specific location (e.g. NMT).

This is definitely something that would be really nice to fix. But the effort required vs. the immediate benefit means that we always end up procrastinating. 🙃